notifications.bs

```diff
@@ -150,114 +158,109 @@ clipped corners.
+  <p class=note><a>Actions</a> are only currently supported for <a>persistent notifications</a>.
 
-  <li><p>If <var>options</var>["{{NotificationOptions/silent}}"] is true and
-  <var>options</var>["{{NotificationOptions/vibrate}}"] <a for=map>exists</a>, then <a>throw</a> a
-  {{TypeError}}.
+ <li><p>If <var>options</var>["{{NotificationOptions/silent}}"] is true and
+ <var>options</var>["{{NotificationOptions/vibrate}}"] <a for=map>exists</a>, then <a>throw</a> a
+ {{TypeError}}.
 
-  <li><p>If <var>options</var>["{{NotificationOptions/renotify}}"] is true and
-  <var>options</var>["{{NotificationOptions/tag}}"] is the empty string, then <a>throw</a> a
-  {{TypeError}}.
+ <li><p>If <var>options</var>["{{NotificationOptions/renotify}}"] is true and
+ <var>options</var>["{{NotificationOptions/tag}}"] is the empty string, then <a>throw</a> a
+ {{TypeError}}.
 
-  <li><p>Set <var>notification</var>'s <a for=notification>data</a> to
-  <a abstract-op>StructuredSerializeForStorage</a>(<var>options</var>["{{NotificationOptions/data}}"]).
+ <li><p>Set <var>notification</var>'s <a for=notification>data</a> to
+ <a abstract-op>StructuredSerializeForStorage</a>(<var>options</var>["{{NotificationOptions/data}}"]).
 
-  <li><p>Set <var>notification</var>'s <a for=notification>title</a> to <var>title</var>.
+ <li><p>Set <var>notification</var>'s <a for=notification>title</a> to <var>title</var>.
 
-  <li><p>Set <var>notification</var>'s
-  <a for=notification>direction</a> to <var>options</var>["{{NotificationOptions/dir}}"].
+ <li><p>Set <var>notification</var>'s <a for=notification>direction</a> to
+ <var>options</var>["{{NotificationOptions/dir}}"].
 
-  <li><p>Set <var>notification</var>'s <a for=notification>language</a> to
-  <var>options</var>["{{NotificationOptions/lang}}"].
+ <li><p>Set <var>notification</var>'s <a for=notification>language</a> to
+ <var>options</var>["{{NotificationOptions/lang}}"].
 
-  <li><p>Set <var>notification</var>'s <a for=notification>origin</a> to the
-  <a>entry settings object</a>'s <a for="environment settings object">origin</a>.
+ <li><p>Set <var>notification</var>'s <a for=notification>origin</a> to <var>settings</var>'s
+ <a for="environment settings object">origin</a>.
 
-  <li><p>Set <var>notification</var>'s <a for=notification>body</a> to
-  <var>options</var>["{{NotificationOptions/body}}"].
+ <li><p>Set <var>notification</var>'s <a for=notification>body</a> to
+ <var>options</var>["{{NotificationOptions/body}}"].
 
-  <li><p>Set <var>notification</var>'s <a for=notification>tag</a> to
-  <var>options</var>["{{NotificationOptions/tag}}"].
+ <li><p>Set <var>notification</var>'s <a for=notification>tag</a> to
+ <var>options</var>["{{NotificationOptions/tag}}"].
 
-  <li><p>Let <var>baseURL</var> be the API base URL specified by the
-  <a>entry settings object</a>. <span class=XXX>Or incumbent?</span>
+ <li><p>Let <var>baseURL</var> be the API base URL specified by <var>settings</var>.
```

Comment:
This could be `<var>settings</var>'s <a for="environment settings object">API base URL</a>`